pkg/console/operator/sync_v400.go

```diff
@@ -59,7 +59,7 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	if svcErr != nil {
 		msg := fmt.Sprintf("%q: %v", "service", svcErr)
 		klog.V(4).Infof("incomplete sync: %v", msg)
-		co.ConditionResourceSyncProgressing(updatedOperatorConfig, msg)
+		co.HandleProgressing(updatedOperatorConfig, "ConsoleServiceSync", svcErr)
```

Comment:
As mentioned above.. let's talk about the enum vs. planeString

Reply:
Sure thing, lets see:
https://github.com/openshift/cluster-authentication-operator/pull/142/files

 I don't believe any of the strings should be reused.  Each one should be specific enough for a single use in one place.